docs/ops/deployment/docker.md

```diff
@@ -70,21 +70,43 @@ and deploys *TaskManagers* on demand so that you do not have to do it manually.
 
 The next chapters describe how to start a single Flink Docker container for various purposes.
 
+Once you've started Flink on Docker, you can access the Flink Webfrontend on [localhost:8081](http://localhost:8081/#/overview) or submit jobs like this `./bin/flink run ./examples/streaming/TopSpeedWindowing.jar`.
```

Comment:
maybe remove the duplicate paragraphs under ´Flink with Docker Compose#Usage` and `Flink with Docker Swarm`

Reply:
I would leave it. I don't think people read the entire page: They just jump to the section they are interested in, thus they won't see this.